components/zoom_admin/actions/list-past-meeting-participants/list-past-meeting-participants.mjs

```diff
@@ -0,0 +1,32 @@
+import zoomAdmin from "../../zoom_admin.app.mjs";
+import get from "lodash/get.js";
+import { paginate } from "../../common/pagination.mjs";
+
+export default {
+  name: "List past meeting participants",
+  description:
+    "List all participants of a past meeting. [See the docs here](https://developers.zoom.us/docs/api/meetings/#tag/meetings/GET/past_meetings/{meetingId}/participants)",
```

Comment:
```suggestion
    "List all participants of a past meeting. [See the documentation](https://developers.zoom.us/docs/api/meetings/#tag/meetings/GET/past_meetings/{meetingId}/participants)",
```
Also if we can replace all instances of "see the docs here" with "see the documentation" in this app, with a quick find+replace, that'd be good for keeping the standard too